### ISSUE TITLE
mdAutocomplete: call textChange on mdAutocompleteCtrl.clear() 

### ISSUE THREAD

**Issue by igdtl** [angular/material#8112]
Currently the `clearValue()` function of `autocompleteController.js` does the following:

1. Assign `0` to `index`
2. Assign `[]` to `matches` property
3. Assign `''` to `searchText` property

I think it should also invoke the `textChange()` callback.

**Comment by clshortfuse** [comment]
If searchText changes, textChange() is eventually called. If the text was already set to '' when it's cleared, nothing will fire.

**Comment by topherfangio** [comment]
@igdtl Does @clshortfuse's answer make sense? If not, can you please provide a Codepen demo showing when you expect it to be called?

**Comment by igdtl** [comment]
After poking around a bit on Codepen I found the following:

1. The bug seems to have been fixed on v1.1.0-rc2
2. It was only triggered when the directive included an md-delay with a "big enough" value (in my case, greater than 12 ms., though it may depend on browser/os/cpu speed)

Here is the Codepen showing the issue:

http://codepen.io/igdtl/pen/LNJGPB

Try changing the md-delay value to 10 or less and see how pressing the clear button "fixes" the bug.
